util/test/test_query.py

```diff
@@ -735,6 +736,40 @@ def test_populateNftAssetNames():
     assert nfts[0].name == "Take a Ballet Lesson"
 
 
+@enforce_types
+def _test_queryPassiveRewards(addresses):
+    chain = brownie.network.chain
+    S_PER_WEEK = 604800
+    fee_distributor = oceanutil.FeeDistributor()
+    oceanutil.OCEANtoken().transfer(
+        fee_distributor.address,
+        toBase18(1000.0),
+        {"from": brownie.accounts[0]},
+    )
+    fee_distributor.checkpoint_total_supply({"from": brownie.accounts[0]})
+    fee_distributor.checkpoint_token({"from": brownie.accounts[0]})
+    # sleep to next week
+    chain.sleep(S_PER_WEEK - chain.time() % S_PER_WEEK + S_PER_WEEK)
+    chain.mine()
+    query.queryPassiveRewards(CHAINID, chain.time(), addresses)
+
+    oceanutil.OCEANtoken().transfer(
+        fee_distributor.address,
+        toBase18(1000.0),
+        {"from": brownie.accounts[0]},
+    )
+    fee_distributor.checkpoint_total_supply({"from": brownie.accounts[0]})
+    fee_distributor.checkpoint_token({"from": brownie.accounts[0]})
+    chain.sleep(S_PER_WEEK * 2)
+    chain.mine()
+    fee_distributor.checkpoint_total_supply({"from": brownie.accounts[0]})
+    fee_distributor.checkpoint_token({"from": brownie.accounts[0]})
+    balances, rewards = query.queryPassiveRewards(CHAINID, chain.time(), addresses)
+    print(balances)
+    print(rewards)
+    assert len(balances) == 90
```

Comment:
I follow the logic of the bootstrap and how it's distributing passive rewards, to then query the balance via subgraph.

But the test is checking for 90 balances.
Will this test pass across all of our workflows?

This seems to be configured against the stress test.

Reply:
Test is not ready yet :face_in_clouds: I added the assert statement to fail the test flow